initializers/groups.yml

```diff
@@ -1,35 +1,9 @@
-## To list all permissions, run:
-## 
-## docker-compose run --rm --entrypoint /bin/bash netbox
-## $ ./manage.py migrate
-## $ ./manage.py shell
-## > from django.contrib.auth.models import Permission
-## > print('\n'.join([p.codename for p in Permission.objects.all()]))
-##
-## Permission lists support wildcards. See the examples below.
-##
-## Examples:
-
-# applications:
+# - name: applications
 #   users:
-#   - technical_user
-# readers:
```

Comment:
These were intentional YAML keys, because YAML keys are only allowed once per YAML document. And there can only be one group name. So a decent IDE will flag a duplicate key, which allows to avoid errors early.

I know it takes some extra effort in the Python code to transform the keyname to the `name` property, but I think that's worth it. Even more so, since it prevents folks from having to touch their existing `groups.yml` file when they migrate to a new version.

Reply:
Corrected.  Not sure why I reverted the existing methodology in the first place.